addons/mass_mailing/models/mailing_contact.py

```diff
@@ -176,6 +194,16 @@ def action_add_to_mailing_list(self):
 
         return action
 
+    def action_view_mailings(self):
+        self.ensure_one()
+        action = self.env["ir.actions.actions"]._for_xml_id("mass_mailing.mailing_trace_action")
+        action["display_name"] = _("Statistics")
+        action["domain"] = [("model", "=", "mailing.contact"), ("res_id", "in", self.ids)]
+        action["views"] = [
+            (self.env.ref("mass_mailing.mailing_trace_view_tree_statistics").id, "list"),
+        ]
+        return action
```

Comment:
couldn't find any usage of the action here
maybe we can return our own action then
```suggestion
        return {
            "name": _("Statistics"),
            "type": "ir.actions.act_window",
            "res_model": "mailing.trace",
            "domain": [("model", "=", "mailing.contact"), ("res_id", "in", self.ids)],
            "views": [(self.env.ref("mass_mailing.mailing_trace_view_tree_statistics").id, "list")],
        }
```